.stylelintrc.json

```diff
@@ -11,6 +11,7 @@
     "a11y/no-outline-none": true,
     "a11y/selector-pseudo-class-focus": true,
     "at-rule-no-unknown": null,
+    "selector-no-vendor-prefix": null,
```

Comment:
Did you find any counter argument/reason we wouldn't want to do this? Just want to make sure I understand the reasoning.

Reply:
This is directly related to placeholder styling. However, I just looked at [caniuse](https://caniuse.com/?search=placeholder) for the base property and it seems like it's pretty broadly supported. We *could* do away with those browser prefixes, which would allow us to re-implement this rule. Thoughts?

Reply:
I just came across this with `position: -webkit-sticky;` in the tableOfContents as well. The stylelint docs note that 

> this rule does not blanketly condemn vendor prefixes. Instead, it uses Autoprefixer's up-to-date data (from caniuse.com) to know whether a vendor prefix should cause a violation or not." 

So if I'm understanding that correctly, it's doing the checks for us, right?

Reply:
OK, thanks for doing the research I should have done :)

Yup, sounds like we should let this do its job. Fixed.